.github/workflows/ci-tests-fabric.yml

```diff
@@ -47,10 +47,14 @@ jobs:
           - {os: "windows-2022", pkg-name: "lightning", python-version: "3.8", pytorch-version: "1.11"}
           - {os: "windows-2022", pkg-name: "lightning", python-version: "3.9", pytorch-version: "1.12"}
           - {os: "windows-2022", pkg-name: "lightning", python-version: "3.10", pytorch-version: "1.13"}
-          # only run PyTorch latest with Python latest
+          # only run PyTorch latest with Python recent
           - {os: "macOS-11", pkg-name: "lightning", python-version: "3.10", pytorch-version: "2.0"}
           - {os: "ubuntu-20.04", pkg-name: "lightning", python-version: "3.10", pytorch-version: "2.0"}
           - {os: "windows-2022", pkg-name: "lightning", python-version: "3.10", pytorch-version: "2.0"}
+          # only run PyTorch latest with Python latest, use Fabric scope to limit dependency issues
```

Comment:
These changes are missing in checkgroup

Reply:
I was thinking of adding them later but fine to add now...